packages/devextreme/testing/tests/DevExpress.ui.widgets/chatParts/messageBubble.tests.js

```diff
@@ -42,6 +43,13 @@ QUnit.module('MessageBubble', moduleConfig, () => {
             assert.strictEqual(this.$element.text(), 'new message text');
         });
 
+        QUnit.test('isDeleted option should delete the message after being set at runtime', function(assert) {
```

Comment:
test for initial render is missed

QUnit.test('bubble should be render with default text if isDeleted is true', function(assert) {
            this.reinit({ isDeleted: true });

            assert.strictEqual(this.$element.text(), messageLocalization.format('dxChat-deletedMessageText'));
        });